test/plugins/test_completion.py

```diff
@@ -23,7 +23,7 @@ def test_completion():
     items = pyls_completions(doc, com_position)
 
     assert len(items) > 0
-    assert items[0]['label'] == 'read'
+    assert items[0]['label'] == 'read()'
```

Comment:
Doesn't make sense to try and deal with 2vs3 (I believe python 3 lets you "inspect" the python * and ** parameters to a function), so you may want to just pick another builtin to test against instead of `sys.stdin.read` !

Reply:
Tried `sys.exit` which also differed, for now I added the version check inline.
I can try a few more later, agree it is clumsy logic to have in the tests.